tests/resolvers/Chat/admins.spec.ts

```diff
@@ -0,0 +1,40 @@
+import "dotenv/config";
+import { admins as adminsResolver } from "../../../src/resolvers/Chat/admins";
+import { connect, disconnect } from "../../helpers/db";
+import type mongoose from "mongoose";
+import { User } from "../../../src/models";
+import { afterAll, beforeAll, describe, expect, it } from "vitest";
+import type { TestChatType } from "../../helpers/chat";
+import { createTestChatMessage } from "../../helpers/chat";
+
+let testChat: TestChatType;
+let MONGOOSE_INSTANCE: typeof mongoose;
+
+beforeAll(async () => {
+  MONGOOSE_INSTANCE = await connect();
+  const userOrgChat = await createTestChatMessage();
+  testChat = userOrgChat[2];
+});
+
+afterAll(async () => {
+  await disconnect(MONGOOSE_INSTANCE);
+});
+
+describe("resolvers -> Chat -> admins", () => {
+  it(`returns user object for parent.users`, async () => {
```

Comment:
The test description (`returns user object for parent.users`) might not be specific enough to describe the exact behavior under test. Update the description to reflect what exactly you are testing (e.g., that the `adminsResolver` is correctly resolving admin users).

```javascript
it(`resolves the correct admin users for the given chat`, async () => {
  const parent = testChat?.toObject();
  if (!parent) {
    throw new Error("Parent object is undefined.");
  }

  const usersPayload = await adminsResolver(parent, {}, {});

  const users = await User.find({
    _id: { $in: testChat?.admins },
  }).lean();

  expect(usersPayload).toEqual(users);
});
```